event-sourcing/subscription-manager/src/main/java/uk/gov/justice/services/event/sourcing/subscription/manager/SubscriptionManagerProducer.java

```diff
@@ -71,14 +68,13 @@ private SubscriptionManager create(final Subscription subscription) {
                 eventSource,
                 interceptorChainProcessor,
                 eventBufferSelector.selectFor(componentName),
-                logger);
+                LOGGER);
```

Comment:
I think this should be LoggerFactory.getLogger(DefaultSubscriptionManager.class) instead of passing the logger for SubscriptionManagerProducer

Reply:
updated